CWE/events/Essential Events 8.txt

```diff
@@ -9648,7 +9648,7 @@ country_event = {
 		SWK = { 
 			government = hms_government1 
 			country_event = 800054 
-			primary_culture = english 
+			primary_culture = british 
```

Comment:
Sarawak should be kept as English as the guy in charge of Sarawak was an Englishman.